test/core/security/secure_endpoint_test.c

```diff
@@ -54,8 +58,8 @@ static grpc_endpoint_test_fixture secure_endpoint_create_fixture_tcp_socketpair(
   grpc_endpoint_add_to_pollset(&exec_ctx, tcp.server, g_pollset);
 
   if (leftover_nslices == 0) {
-    f.client_ep =
-        grpc_secure_endpoint_create(fake_read_protector, tcp.client, NULL, 0);
+    f.client_ep = grpc_secure_endpoint_create(
+        fake_read_protector, fake_read_grpc_protector, tcp.client, NULL, 0);
```

Comment:
Shouldn't we run all secure endpoint tests twice, once with the old-style frame protector, and again with the zero-copy frame protector?  Otherwise, we're not testing both branches of the code.

Reply:
DONE.